src/main/java/seedu/address/logic/commands/DeletePersonCommand.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import picocli.CommandLine;
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Deletes a person identified using its displayed index from the address book.
+ */
+@CommandLine.Command(name = "person")
+public class DeletePersonCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete person";
+
+    public static final String MESSAGE_USAGE =
+            COMMAND_WORD + ": Deletes the person identified by the index number used in the displayed person list.\n"
+                    + "Parameters: INDEX (must be a positive integer)\n" + "Example: " + COMMAND_WORD + " 1";
```

Comment:
We probably need a consistent way to differentiate `MEMBER_INDEX` (from the current `teamMembersList`) and the overall contacts list (`PERSON_INDEX` or `CONTACT_INDEX`)
```suggestion
                    + "Parameters: PERSON_INDEX (must be a positive integer)\n" + "Example: " + COMMAND_WORD + " 1";
```